src/main/java/seedu/address/model/assignment/Schedule.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.assignment;
+
+public class Schedule {
+    private String dummyDeadline = "01-01-2000 0101";
+    private boolean schedule;
+    private Deadline suggestedStartTime;
+    private Deadline suggestedEndTime;
```

Comment:
Should these variables be final instead?

Reply:
Changed to Final! thanks